<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>581</width>
      <height>450</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Valores</string>
     </attribute>
     <widget class="QLabel" name="labelPic">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>-20</y>
        <width>350</width>
        <height>200</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/imagem/RLC.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>170</y>
        <width>494</width>
        <height>214</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_12">
             <item>
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>Tensão da Fonte (V):            </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="impTensao"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_13">
             <item>
              <widget class="QLabel" name="label_11">
               <property name="text">
                <string>Frequência (Hz):                   </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="impFreq"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_14">
             <item>
              <widget class="QLabel" name="label_12">
               <property name="text">
                <string>Passo de Calculo (s):      </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="passoAtual">
               <item>
                <property name="text">
                 <string/>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>1.0e-3</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>5.0e-4</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_15">
             <item>
              <widget class="QLabel" name="label_13">
               <property name="text">
                <string>Tempo de Simulação (s):</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="tempoAtual">
               <item>
                <property name="text">
                 <string/>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>0.05</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>0.1</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>0.5</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>1</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_16">
             <item>
              <widget class="QLabel" name="label_14">
               <property name="text">
                <string>Resistência do Filtro (ohms):  </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="impResA"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_17">
             <item>
              <widget class="QLabel" name="label_15">
               <property name="text">
                <string>Indutância do Filtro (H):         </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="impIndA"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_18">
             <item>
              <widget class="QLabel" name="label_16">
               <property name="text">
                <string>Resistência da Carga (ohms):</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="impResB"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_19">
             <item>
              <widget class="QLabel" name="label_17">
               <property name="text">
                <string>Indutância da Carga (H):       </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="impIndB"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_20">
             <item>
              <widget class="QLabel" name="label_18">
               <property name="text">
                <string>Capacitância (F):                   </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="impCap"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="btnSimular">
         <property name="text">
          <string>SIMULAR</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Simulação</string>
     </attribute>
     <widget class="QTabWidget" name="tab_8">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>571</width>
        <height>421</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Fonte</string>
       </attribute>
       <widget class="QCustomPlot" name="customPlotA" native="true">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>551</width>
          <height>381</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>Filtro Indutivo</string>
       </attribute>
       <widget class="QCustomPlot" name="customPlotB" native="true">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>551</width>
          <height>381</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Filtro Capacitivo</string>
       </attribute>
       <widget class="QCustomPlot" name="customPlotC" native="true">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>551</width>
          <height>381</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_7">
       <attribute name="title">
        <string>Carga Elétrica</string>
       </attribute>
       <widget class="QCustomPlot" name="customPlotD" native="true">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>551</width>
          <height>381</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Resultados</string>
     </attribute>
     <widget class="QTableWidget" name="tabela">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>10</y>
        <width>360</width>
        <height>350</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>Vetor</string>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Módulo</string>
       </property>
       <property name="font">
        <font>
         <family>MS Shell Dlg 2</family>
         <pointsize>8</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Ângulo de Fase</string>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </column>
     </widget>
     <widget class="QSplitter" name="splitter">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>370</y>
        <width>571</width>
        <height>41</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QPushButton" name="btnOrdenarModulo">
       <property name="text">
        <string>Ordenar Por Módulo</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnOrdenarAngulo">
       <property name="text">
        <string>Ordenar Por Ângulo de Fase</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnLimparTabela">
       <property name="text">
        <string>Limpar Tabela</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="Menu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionAbrir"/>
    <addaction name="actionSalvar"/>
    <addaction name="separator"/>
    <addaction name="actionSair"/>
   </widget>
   <widget class="QMenu" name="Ajuda">
    <property name="title">
     <string>Ajuda</string>
    </property>
    <addaction name="actionConsultar_manual"/>
   </widget>
   <addaction name="Menu"/>
   <addaction name="Ajuda"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbrir">
   <property name="icon">
    <iconset resource="imagem.qrc">
     <normaloff>:/imagem/icon_abrir.png</normaloff>:/imagem/icon_abrir.png</iconset>
   </property>
   <property name="text">
    <string>Abrir</string>
   </property>
  </action>
  <action name="actionSalvar">
   <property name="icon">
    <iconset resource="imagem.qrc">
     <normaloff>:/imagem/icon_salvar.png</normaloff>:/imagem/icon_salvar.png</iconset>
   </property>
   <property name="text">
    <string>Salvar</string>
   </property>
  </action>
  <action name="actionSair">
   <property name="icon">
    <iconset resource="imagem.qrc">
     <normaloff>:/imagem/icon_sair.png</normaloff>:/imagem/icon_sair.png</iconset>
   </property>
   <property name="text">
    <string>Sair</string>
   </property>
  </action>
  <action name="actionConsultar_manual">
   <property name="icon">
    <iconset resource="imagem.qrc">
     <normaloff>:/imagem/icon_ajuda.png</normaloff>:/imagem/icon_ajuda.png</iconset>
   </property>
   <property name="text">
    <string>Consultar manual</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="imagem.qrc"/>
 </resources>
 <connections/>
</ui>
